duke/src/main/java/Command.java

```diff
@@ -0,0 +1,18 @@
+import java.text.ParseException;
+
+abstract class Command {
+	protected String command;
+	protected String remainingCommand;
```

Comment:
It is unclear what the purpose of this remainingCommand variable is. Perhaps a short javadoc comment would help?